### ISSUE TITLE
Error messages for OpenIDConnect in the otobo.log are incomplete

### ISSUE THREAD

**Issue by nobbiew** [RotherOSS/otobo#2652]
In the file Kernel/System/Auth/OpenIDConnect.pm lines 128 and 129 the string for the $Message variable is set incorrectly. This means the information is lost.
The same applies to the Kernel/System/CustomAuth/OpenIDConnect.pm file

Correct Lines must be
       ```
$Message .= $ParamObject->GetParam( Param => 'error_description' ) ? "\n". $ParamObject->GetParam( Param => 'error_description' ) : '';
$Message .= $ParamObject->GetParam( Param => 'error_uri' )         ? "\nsee".  $ParamObject->GetParam( Param => 'error_uri' )     : '';
```

**Comment by svenoe** [comment]
In rel-10_1 these lines are 132 and 133 and look like your suggestions (disregarding some white space):
https://github.com/RotherOSS/otobo/blob/45fed1b1577bf65d5405a8c3c0d5a3160b1cb405/Kernel/System/Auth/OpenIDConnect.pm#L132-L133

Could you please check whether your file is an old version and if not specify what exactly your suggested change is?

**Comment by nobbiew** [comment]
Oh, sorry

seems it was fixed with Issue #2301. I tested with dockerimage rel-10_1_7 and didn´t check this in GitHub. So issue can be closed.
